src/cdk/tree/tree.md

```diff
@@ -176,3 +176,10 @@ tree how to uniquely identify nodes to track how the data changes with each upda
 ```html
 <cdk-tree [dataSource]="dataSource" [treeControl]="treeControl" [trackBy]="trackByFn">
 ```
+
+Make sure to also provide a similar `trackBy` function on the `TreeControl`. For example:
+```ts
+const treeControl = new NestedTreeControl<FoodNode, string>(node => node.children, {
+  trackBy: (node) => node.id
+});
+```
```

Comment:
Added a small explanation since this wasn't really clear from the docs. Most CDK components only rely the `trackBy` function to be configured on the component, not on the control. I believe this might be an issue in the design, but I don't see a way to improve/fix this without introducing breaking changes.